Code/Tools/AssetProcessor/native/resourcecompiler/RCBuilder.cpp

```diff
@@ -557,7 +512,7 @@ namespace AssetProcessor
             AZ_TracePrintf(AssetProcessor::ConsoleChannel,
                 "Job request for %s in builder %s missing job parameters.",
                 request.m_sourceFile.c_str(),
-                BUILDER_ID_RC.GetId().toUtf8().data());
+                request.m_builderGuid.ToString<AZStd::string>().c_str());
```

Comment:
FYI: There's a .ToFixedString now that you can use, it's slightly cleaner looking and might even avoid an allocation.